tmt/steps/execute/internal.py

```diff
@@ -192,15 +203,15 @@ def log(key, value=None, color=None, shift=1, level=1):
         self.verbose(
             f"{duration} {test.name} [{progress}]{timeout}", shift=shift)
 
-    def check(self, test):
+    def check(self, test: Test) -> Any:
```

Comment:
Follow the dots: `check()` calls either `self.check_beakerlib()` or `self.check_shell()`, what do these two methods return? Can we be more specific than `Any`? I believe we can, inspect those two methods.